.github/workflows/ci_python.yml

```diff
@@ -132,28 +132,33 @@ jobs:
             echo "::endgroup::"
             deactivate
           done
-      - name: Setup QT
-        # Needed by sphinx-social-cards.
-        # https://github.com/2bndy5/sphinx-social-cards/blob/main/.github/workflows/build.yml#L54
+      - name: Lint reference docs
         run: |
-          sudo apt-get install -y libgl1-mesa-dev libxkbcommon-x11-0
-          echo "QT_QPA_PLATFORM=offscreen" >> "$GITHUB_ENV"
+          poetry env use 3.11
+          source $(poetry env info --path)/bin/activate
+          poetry install --with=docs
+          pip install ${WHEEL} --force-reinstall
```

Comment:
@bjchambers should I do this in each sub-task, or is doing it here enough: https://github.com/kaskada-ai/kaskada/pull/799/files#diff-47057ea60e76362be7390920540cdc8bfa927006bad30cd6f59189f05daab815L125

Reply:
Each task in the job runs in the same environment. So istalling it here should mean that it is available to `Set up quarto`, etc. However, you then `deactivate` the python environment, so it's unclear if that will persist.